.github/no-response.yml

```diff
@@ -0,0 +1,4 @@
+daysUntilClose: 14
```

Comment:
This will auto-close issues. Is this something we want to do?

Reply:
Definitely not. Good catch.